integration_tests/tests/forseti/controls/scanner.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+control 'Scanner' do
+
+
+   describe "Run scanner" do
+
+       before :context do
+           command("forseti inventory purge 0").result
+           command("forseti inventory create").result
```

Comment:
Why is the inventory purged and created again?  Can the existing one be re-used?

Reply:
This is the first time inventory and model are created in `Scanner` control. It is needed for scanner to run successfully. If you're referring to re-using this from other controls, it is doable, but a new library needs to be created/module needs to be added. Do you think it's worth the effort?